src/templates/ratings/add.html

```diff
@@ -1,6 +1,6 @@
 {% macro review_prompt(slug, isModal, existingReview) %}
```

Comment:
unfortunately, we lost any standards we had for camelCase vs under_scores when fireplace was started.